modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/AuditBarChart.js

```diff
@@ -32,7 +33,19 @@ import {
 import {BAR_CHART, COLORS, DEFAULT_COLOR} from '../utils/constants';
 import {shortenNumber} from '../utils/shortenNumber';
 
-export default function AuditBarChart({rtl, vocabularies}) {
+const ESCAPE_KEYS = new Set(['Esc', 'Escape']);
+
+const handleKeydown = (event) => {
+	const resetBarsCategoryFiltersURL = new URLSearchParams(
+		window.location.href
+	).get('resetBarsCategoryFiltersURL');
+
+	if (ESCAPE_KEYS.has(event.key) && resetBarsCategoryFiltersURL) {
```

Comment:
Since we are supporting latest Edge, we can just get rid of checking for `Esc`. In latest edge, the key is `Escape` since latest edge is just chromium based. So really for this, we can remove the Set and just check `event.key === 'Escape'`.

It's only in non-chromium edge and IE that we would need to check for `Esc`, and we don't need to support those.

Reply:
As @wincent said [here](https://github.com/liferay-frontend/liferay-portal/pull/798#discussion_r577735309), we can use `URLSearchParams` for IE too because we have a polyfill. So I let the `Esc` check and support IE 😄

Reply:
@dgarciasarai is supporting IE an explicit requirement for this? If we need to explicitly require it, then we can leave it in. But if we are just including it "just in case" then I don't think we should include it.


@wincent am I mistaken in that we no longer need to explicitly support IE?

Reply:
AFAIK, Content Dashboard was released in 7.3 and these changes may be backported. Also, in that portal version we still support IE, right? IE it's not supported in 7.4. See https://liferay.slack.com/archives/CJ1ERLHUZ/p1599774562070300.

Reply:
> AFAIK, Content Dashboard was released in 7.3 and these changes may be backported. Also, in that portal version we still support IE, right? I think that Liferay Portal 7.4 won't support IE anymore. See this announcement. Thanks!

Hey @dgarciasarai, back when we announced IE was not supported anymore in `master`, we took very specific steps to update our codebase with that in mind. We managed to get rid of plenty of hacks and workarounds that were there simply to support that browser. We've even recently updated all of our JSPs to use arrow functions directly (a thing we won't be backporting) but that makes our whole codebase stay in sync and target the specific browsers we're using.

I agree with @bryceosterhaus that "just in case" is not a strong enough motivation to do something. We should have better railguards for backports. Backporting something shouldn't simply be an administrative issue (moved this commit from here to there) but rather a product decision (moved, verified, tested...). Backporting something is more than just moving the code back to older branches, and making sure the code runs in the older version platforms should be part of it.

This being said, if you have a real indication that this code is indeed going to be backported, there's nothing wrong in making your life easier. You can make an initial version (like this) and then add one extra commit removing whatever unnecessary support you have. That way, you simply need to avoid cherry-picking one commit. The same can be done the other way around. Just take the time to document (in a commit, or in the JIRA issue) what would be necessary for this to be effectively backported so whoever gets to do that eventually has access to that information.

Reply:
I just confirm with product and we are NOT going to backport this to 7.3.x. So I'm going to apply the last feedback from @bryceosterhaus and check `event.key === 'Enter'` and remove the `Set` I previously created. Thanks @jbalsas for the explanation. I'll take into account commits structure in case a functionality will be backported ;)

Reply:
@jbalsas thanks for the thorough explanation! Backports have never been super clear to me and so thinking through them like product decisions is a helpful way to think about them.

And thanks for addressing this @dgarciasarai, I think more of my concern here was my own lack of knowledge in our backport and support structure for various browsers. I hadn't seen any code targeted at IE in awhile and so it raised a flag for me. Thanks for checking with product and working through it though!

Reply:
That's code review! 😄 Thanks! ❤️

Reply:
Sorry if my comment about IE misled you: I wasn't meaning to say "you have to support IE"; rather, I meant "you don't have to worry about IE" (on `master`, because we don't need to support it, and on older branches, because we polyfill).